src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -1722,7 +1723,7 @@ public void testDeleteOrderLineById500FromStorageOnGetPoLine() {
   public void testDeleteOrderLineById500FromStorageOnSubObjectDeletion() {
```

Comment:
This test case is exactly same as the above test case, it would be nice to simulate the deletion of alerts

Reply:
Removed redundant test